src/implementations/twig/components/text-area/text-area.story.js

```diff
@@ -1,24 +1,64 @@
 import { withKnobs } from '@storybook/addon-knobs';
 import { withNotes } from '@ecl/storybook-addon-notes';
-import { getExtraKnobs, getFormKnobs } from '@ecl/story-utils';
+import { getFormControls } from '@ecl/story-utils';
 import withCode from '@ecl/storybook-addon-code';
-import specData from '@ecl/specs-component-text-area/demo/data--default';
+import demoData from '@ecl/specs-component-text-area/demo/data--default';
+
 import textArea from './text-area.html.twig';
 import notes from './README.md';
 
-const prepareTextArea = (data) => {
-  getFormKnobs(data);
-  getExtraKnobs(data);
+const getArgTypes = (data) => {
+  return {
+    ...getFormControls(data),
+    width: {
```

Comment:
I added width control to the utils, see pending https://github.com/ec-europa/europa-component-library/pull/1842 or https://github.com/ec-europa/europa-component-library/pull/1844